src/algebra/gcd_monoid/basic.lean

```diff
@@ -557,9 +557,11 @@ begin
       mul_dvd_mul_iff_left h1, mul_dvd_mul_iff_right h2, and_comm] }
 end
 
-lemma dvd_lcm_left [gcd_monoid α] (a b : α) : a ∣ lcm a b := (lcm_dvd_iff.1 dvd_rfl).1
+lemma dvd_lcm_left [gcd_monoid α] (a b : α) : a ∣ lcm a b :=
+(lcm_dvd_iff.1 (@dvd_rfl _ _ (lcm a b))).1
```

Comment:
Typechecking fails to infer the correct implicit arguments for `dvd_rfl` now, which is unfortunate.

Reply:
you can use `dvd_refl` here instead; this is unfortunate, though. is this the only place where this breaks?

Reply:
Yes, this was the only breakage.